src/v2/models/AppState.js

```diff
@@ -87,6 +87,10 @@ export default Model.extend({
     }
   },
 
+  getPollInterval() {
```

Comment:
Seems not worth add function in appState given it's only used for `PollView` rather adds to `PollView` for now.